changelogs/fragments/24-use-podman-in-integration-test.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
Please remove this changelog, as it's related to tests not to code. Let's keep in changelog only things related to whatever is released to user.